wallets/keplr/src/utils/selectors/createDataTestSelectors.ts

```diff
@@ -0,0 +1,7 @@
+export const createDataTestSelector = (dataTestId: string) => {
```

Comment:
Same as above:

I rolled back a shared package, which this would fall into. We can identify these shared functions and move them to a new package in a later issue. For now want to make sure the library is self contained.